netlink-packet-route/src/rtnl/tc/nlas/qdisc/prio.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+//#[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
Should this file go? Or are you planning to add more?

Reply:
Remove it for now